src/ip2lib_4.h

```diff
@@ -1,33 +1,147 @@
+ * - 16 longitude of last grid point, octets 60-63
+ * - 17 i-direction increment, octets 64-67
+ * - 18 number of parallels between pole and equator, octets 68-71
+ * - 19 scanning mode, octet 72
+ * polar stereographic azimuthal:
+ * - 8  number of points along x-axis, octets 31-34
+ * - 9  number of points along y-axis, octets 35-38
+ * - 10 latitude of first grid point, octets 39-42
+ * - 11 longitude of first grid point, octets 43-46
+ * - 12 resolution and component flags, octet 47
+ * - 13 true latitude, octets 48-51
+ * - 14 orientation longitude, octets 52-55
+ * - 15 x-direction grid length, octets 56-59
+ * - 16 y-direction grid length, octets 60-63
+ * - 17 projection center flag, octet 64
+ * - 18 scanning mode flags, octet 65
+ * rotated equidistant cyclindrical:
+ * - 8  number of points along a parallel, octs 31-34
+ * - 9  number of points along a meridian, octs 35-38
+ * - 10 basic angle of initial production domain, octets 39-42
+ * - 11 subdivisions of basic angle, octets 43-46
+ * - 12 latitude of first grid point, octets 47-50
+ * - 13 longitude of first grid point, octets 51-54
+ * - 14 resolution and component flags, octet 55
+ * - 15 latitude of last grid point, octets 56-59
+ * - 16 longitude of last grid point, octets 60-63
+ * - 17 i-direction increment, octets 64-67
+ * - 18 j-direction increment, octets 68-71
+ * - 19 scanning mode, octet 72
+ * - 20 latitude of southern pole of projection, octets 73-76
+ * - 21 longitude of southern pole of projection, octets 77-80
+ * - 22 angle of rotation of projection, octs 81-84
+ * @param igdtlen number of elements of the grid definition
+ * template array. Corresponds to the gfld%igdtlen component of the
+ * ncep g2 library gridmod data structure.
+ * @param iopt option flag
+ * - 0 to compute earth coords of all the grid points
+ * - 1 to compute earth coords of selected grid coords
+ * - -1 to compute grid coords of selected earth coords
+ * @param npts integer maximum number of coordinates
+ * @param fill real fill value to set invalid output data (must
+ * be impossible value; suggested value: -9999.)
+ * @param xpts real (npts) grid x point coordinates if iopt>0
+ * @param ypts real (npts) grid y point coordinates if iopt>0
+ * @param rlon real (npts) earth longitudes in degrees e if
+ * iopt<0 (acceptable range: -360. to 360.)
+ * @param rlat real (npts) earth latitudes in degrees n if
+ * iopt<0 (acceptable range: -90. to 90.)
+ * @param nret number of valid points computed (-1 if
+ * projection unrecognized)
+ * @param crot (npts) clockwise vector rotation cosines
+ * @param srot (npts) clockwise vector rotation sines
+ * (ugrid=crot*uearth-srot*vearth, vgrid=srot*uearth+crot*vearth)
+ * @param xlon (npts) dx/dlon in 1/degrees
+ * @param xlat (npts) dx/dlat in 1/degrees
+ * @param ylon (npts) dy/dlon in 1/degrees
+ * @param ylat (npts) dy/dlat in 1/degrees
+ * @param area (npts) area weights in m**2 (Proportional to the 
```

Comment:
Here is an explanation of what the area weights are proportional to.

Reply:
Where? To what do you refer?